index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Juliana DeCarvalho Sutherland, Software Engineer Student + Full Stack Developer</title>
+    <link href="https://fonts.googleapis.com/css?family=Caveat|Montserrat:400,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+    <main>
+      <!-- ***************  ABOUT / PROFILE  **************** -->
+      <header>
+        <div class="content-wrap">
+          <h1>Juliana DeCarvalho Sutherland</h1>
+          <h2>Software Engineer Student & Environmental Scientist</h2>
+
+          <p>Full-stack developer with a love for new technology, a curious mind for continuous learning, and a passion for solving complex problems in a creative way. I value diversity, inclusion, precision, and attention to detail. In my previous life, I worked with scientists across the globe to create open-source science and scientific materials accessible to all. This allowed me to build a strong foundation in collaboration, relationships, and communication skills.</p>
+          <p>I’m also exploring more creative pursuits with my photography & the love for the outdoors. <a href="https://www.instagram.com/j.by.the.bay/" target="_blank">j.by.the.bay</a></p>
+        </div>
+      </header>
+
+      <!-- ************ PROJECTS / PORTFOLIO  ************** -->
+      <section class="projects">
+        <div class="content-wrap divider">
+          <h2>Featured Projects</h2>
+          <p>View selected projects below. More information & other projects can be found at <a href="https://github.com/julianadecarvalho"> Juliana Github Repo</a>.</p>
+
+          <!-- Project 1 -->
+          <section class="project-item">
+            <img src="images/david-chus.png" alt="David Chu's China Bistro website">
+            <h3>David Chu's China Bistro</h3>
+            <p>Final project for the HTML, CSS, & JavaScript for Web Developers from Johns Hopkins University on Coursera. Built a fully functional web app that used JavaScript and utilized Ajax.</p>
+            <a class="btn" href="https://julianadecarvalho.github.io/module5-solution/" target="_blank">Final Site</a>
+            <a class="btn" href="https://github.com/julianadecarvalho/johns-hopkins-fullstack-cousera-assignments" target="_blank">GitHub Repo</a>
+          </section>
+
+          <!-- Project 2 -->
+          <section class="project-item">
+            <img src="images/david-chus.png" alt="Video Store CLI">
+            <h3>Video Store CLI</h3>
+            <p>RESTful routes</p>
+            <a class="btn" href="https://julianadecarvalho.github.io/module5-solution/" target="_blank">Final Site</a>
+            <a class="btn" href="https://github.com/julianadecarvalho/johns-hopkins-fullstack-cousera-assignments" target="_blank">GitHub Repo</a>
```

Comment:
These links and images are still for David Chu's restaurant. Not sure if this was just filler for now until you have more time but thought I'd point it out just in case.